src/java.base/share/classes/java/lang/Runtime.java

```diff
@@ -156,6 +156,11 @@ private Runtime() {}
      * <p> The {@link System#exit(int) System.exit} method is the
      * conventional and convenient means of invoking this method.
      *
+     * @implNote
+     * If the {@link System#getLogger(String) System.getLogger("java.lang.Runtime")}
```

Comment:
It may read better to use a textual description for the link, like "the system logger for `java.lang.Runtime`".

Reply:
ok, will fix.